dts/arm/atmel/sam4e.dtsi

```diff
@@ -241,6 +241,14 @@
 			peripheral-id = <8>;
 			status = "disabled";
 		};
+
+		rtc: rtc@400E1860 {
+			compatible = "atmel,sam-rtc";
+			reg = <0x400E1860 0x100>;
+			interrupts = <2 0>;
+			peripheral-id = <2>;
+			status = "disabled";
+		};
```

Comment:
I'm not sure if you can use UpperCase here `400E1860`

Reply:
changed to lower case